src/main/java/uk/gov/hmcts/reform/sscs/model/ServiceRoleMapping.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.reform.sscs.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import uk.gov.hmcts.reform.sscs.utility.TrimStringFields;
+
+
+@Builder
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class ServiceRoleMapping extends CaseWorkerDomain {
+
+    @MappingField(columnName = "Service ID", position = 1)
+    @NotEmpty
+    @JsonProperty(value = "service_code")
+    @JsonDeserialize(using = TrimStringFields.class)
+    String serviceId;
```

Comment:
Same access modifier question here and below

Reply:
this is a copy/paste class from the reference API - one day these models should be shared between projects, so for now I decided not to touch it:
https://github.com/hmcts/rd-caseworker-ref-api/blob/master/src/main/java/uk/gov/hmcts/reform/cwrdapi/client/domain/ServiceRoleMapping.java#L27